frontend/app/globals.css

```
@@ -116,8 +116,16 @@
 }
 
+@property --scroll-thumb-alpha {
+  syntax: '<number>';
+  inherits: true;
+  initial-value: 0;
+}
+
 html {
   overflow-x: hidden;
+  --scroll-thumb-alpha: 0;
+  transition: --scroll-thumb-alpha 0.3s ease;
```

Stylelint (17.3.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)